### PR TITLE
Added blockchain books

### PR DESCRIPTION
**Pull request by SomTambe**
Added two books about BlockChain

## What does this PR do?
Add Resources
## For resources
BlockChain books.
### Description 
Regarding issue #3223 . I have added two great blockchain guides.
### Why is this valuable (or not)
This resource did not exist before.
### How do we know it's really free?
One is an official pdf, the other one is the official guide.
### For book lists, is it a book?
N.A.
### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-courses-en.md [hunk 1]
```diff
@@ -97,6 +98,9 @@
 * [Bento Shell Track](https://bento.io/topic/shell) (Bento)
 * [Shell Scripting Tutorial](https://www.youtube.com/playlist?list=PL7B7FA4E693D8E790)
 
+### BlockChain
+* [Bitcoin and Cryptocurrency Technologies](https://www.lopp.net/pdf/princeton_bitcoin_book.pdf) - Bitcoin and Cryptocurrency Technologies
```
Comment (eshellman) [inline free-courses-en.md]: duplicate of book in ###language_agnostic####misc; why do you think "lopp.net" is an official source?

File: free-courses-en.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 * [Angular](#angular)
 * [Assembly](#assembly)
 * [Bash / Shell](#bash--shell)
+* [BlockChain](#blockchain)
```
Comment (eshellman) [inline free-courses-en.md]: "BlockChain" isn't a language. It's high time we created a ###language_agnostic####blockchain category, there are already two books on bitcoin that could be moved from ####misc
Wikipedia has "blockchain" as a single word. I don't see it in intercap style much.

File: free-courses-en.md [hunk 3]
```diff
@@ -97,6 +98,9 @@
 * [Bento Shell Track](https://bento.io/topic/shell) (Bento)
 * [Shell Scripting Tutorial](https://www.youtube.com/playlist?list=PL7B7FA4E693D8E790)
 
+### BlockChain
+* [Bitcoin and Cryptocurrency Technologies](https://www.lopp.net/pdf/princeton_bitcoin_book.pdf) - Bitcoin and Cryptocurrency Technologies
+* [Official BlockChain guide](https://bitcoin.org/en/blockchain-guide)
```
Comment (eshellman) [inline free-courses-en.md]: good resource.

Comment (WinnerIT) [reply free-courses-en.md:103]: The url was changed. New link is [here](https://developer.bitcoin.org/devguide/index.html)

Comment (eshellman) [reply free-courses-en.md:103]: Maybe give this PR a fresh start?

Comment (SomTambe) [reply free-courses-en.md:103]: I think we shall do that. I will close this PR, and work on a new one. Also I am a bit tight on time these days, I will work on enhancements not immediately. I hope you understand 😄  .

**Comment by SomTambe** [comment]
I shall close this PR as mentioned above. Thanks for all the suggestions everybody has given.

**Pull request closed without merging**